code-challenges.js

```diff
@@ -30,25 +53,69 @@ const temperature2 = 350
 
 const temperature3 = 212
 // Expected output: "212 is at boiling point"
-
+const bp = 212
+
+const boilingPoint = (t) => {
+  if (t > bp ) {
+    return `${t} is above boiling point`
+  }
+  else if (t === bp) {
+    return `${t} is at boiling point`}
+  else if (t < bp)
+    return `${t} is at boiling point`
+  else {
+    return "Something went wrong"
+  }
+  }
+  console.log(boilingPoint(temperature1)) // Output: "42 is below boiling point"
+  console.log(boilingPoint(temperature2)) // Output: "350 is above boiling point"
+  console.log(boilingPoint(temperature3)) // Output: "212 is at boiling point"
 // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
 // Pseudo code:
+// 1. First I must define the arrays. 
+// 2. Once I have done this I will use the .concat method to define each array into a new array called combinedRuns.
```

Comment:
Careful of your wording, you are not defining each array into a new array, but you are concatenating them together.